src/users/__tests__/deleteFavorite.test.ts

```diff
@@ -0,0 +1,45 @@
+import * as jwt from "jsonwebtoken"
+
+import { IEnvironment } from "../../types"
+import resource from "../index"
+
+const emptyPromise = (val: any) => new Promise((resolve, _) => resolve(val))
+const mockFetch = jest.fn(() => emptyPromise(emptyPromise({})))
+const token =
+  "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzUxMiJ9.eyJ1c2VyX2lkIjoxMDAsImlhdCI6MTU3NDA4NjEwOCwiZXhwIjoxNTg4NDg2MTA4LCJtaGJvX2FjY2Vzc19rZXkiOiJlODFlMzAwZDE3MWYwNjgxNzYxYjEzY2NhNGNkMjRlNyJ9.EyHqEJ2YerWmrbKPtDmUttrb6bnKvf3Ie0VbNjT4MDIrT9B8mQuEwbQ5ue3zvzJ9CvCuHUVXDHRVRFeaN6UhQg"
+
+const users = resource(
+  {
+    token
+  },
+  IEnvironment.Development,
+  mockFetch
+)
+
+describe("the users resource #deleteFavorite", () => {
+  beforeEach(() => {
+    users.deleteFavorite(6)
+  })
+
+  it("should make 1 fetch request", () => {
+    expect(mockFetch.mock.calls.length).toBe(1)
+  })
+
+  it("should call the search endpoint with correct data in body", () => {
```

Comment:
Not super important, but can we  just update the description from the original template of "search endpoint" and make it specific to delete.